content/release-10-17-0/applications-10-17-0-bundle/10-17-0-0.md

```diff
@@ -0,0 +1,72 @@
+---
+weight: 40
+title: Release 10.17.0.0
+layout: bundle
+---
+
+<!--10.16.1.0 - 10.16.49.0-->
+
+### Improvements
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+</tbody></table></div>
+
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Cockpit</td>
+<td>The "Welcome" widget in the Cockpit application is now reliably displaying the <b>Register device</b> link as long as the "Devicemanagement" application is subscribed to the tenant.</td>
+<td>MTM-49324</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>The alert message which shows up if a widget still references a deleted device has been improved to provide more clarity for the user. This change has been implemented in the following widgets: "Data points list", "Radial gauge", "Linear gauge", "Silo" and "Pie chart".</td>
+<td>MTM-45516</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>"Help and Service" widget icons are no longer missing after a hard refresh.</td>
+<td>MTM-47547</td>
+</tr>
+
+
+<tr>
+<td>Java SDK</td>
+<td>Fixed an issue with the <code>AlarmRepresentation</code> comparison in equals and hashcode methods. Now, instead of comparing the full <code>ManagedObjectRepresentation</code> as source of alarms, we only compare the managed objects ID. There is no change in <code>ManagedObjectRepresentation</code>.</td>

```

Comment:
```suggestion
<td>Fixed an issue with the <code>AlarmRepresentation</code> comparison in the <code>equals</code> and <code>hashcode</code> methods. Now, instead of comparing the full <code>ManagedObjectRepresentation</code> as the source of alarms, we only compare the managed objects ID. There is no change in <code>ManagedObjectRepresentation</code>.</td>

```